auth/src/main/java/com/firebase/ui/auth/ui/account_link/SaveCredentialsActivity.java

```diff
@@ -89,7 +89,7 @@ protected void onStart() {
 
     @Override
     public void onConnected(@Nullable Bundle bundle) {
-        if (mEmail == null) {
+        if (mEmail == null || mEmail.equals("")) {
```

Comment:
We should investigate under what circumstances a null / empty email address ever arrives here. I agree with sam however that `if (TextUtils.isEmpty(mEmail)` is a more descriptive check here.